apps/alphabet_mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,25 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <uses-permission android:name="android.permission.INTERNET" />
+
+    <application
+      android:name=".MainApplication"
```

Comment:
Be sure you are using actually names \ identifiers for the Alphabet app right away. These can be annoying to hunt down when you're ready to publish.